docs/fsm-diagrams.md

```diff
@@ -24,6 +24,32 @@ Terminate:::terminate --> Terminate:::terminate
 
 ```
 
+## [BallPlacementPlayFSM](/src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.h)
+
+```mermaid
+
+stateDiagram-v2
+classDef terminate fill:white,color:black,font-weight:bold
+direction LR
+[*] --> StartState
+StartState --> AlignPlacementState : [!shouldKickOffWall]\n<i>alignPlacement</i>
+StartState --> KickOffWallState : [shouldKickOffWall]
+KickOffWallState --> KickOffWallState : [!kickDone && shouldKickOffWall]\n<i>kickOffWall</i>
+KickOffWallState --> KickOffWallState : [kickDone]
+KickOffWallState --> AlignPlacementState : [!kickDone]
+AlignPlacementState --> KickOffWallState : [shouldKickOffWall]
+AlignPlacementState --> AlignPlacementState : [!alignDone]\n<i>alignPlacement</i>
+AlignPlacementState --> PlaceBallState : [alignDone]
+PlaceBallState --> PlaceBallState : [!ballPlaced]\n<i>placeBall</i>
+PlaceBallState --> WaitState : [ballPlaced]\n<i>startWait</i>
+WaitState --> WaitState : [!waitDone]
+WaitState --> RetreatState : [waitDone]
+RetreatState --> Terminate:::terminate : [retreatDone && ballPlaced]
+RetreatState --> RetreatState : [!ballPlaced]\n<i>placeBall</i>
+RetreatState --> RetreatState : [ballPlaced]\n<i>retreat</i>
```

Comment:
I'm not sure if this is related to you pr, but when I go to https://github.com/germchow/Software/blob/jeremy/clean-git/docs/fsm-diagrams.md all the diagrams have plaintext `<i>` and `<i/>` tags included for all the diagrams - is this just a render issue on my end?

Reply:
looks like its the same issue on master